README.md

```diff
@@ -459,6 +459,11 @@ Let's create a `documents` and `document_sections` table to store our processed
     );
     ```
 
+    _Note: Since the video was published, `on delete cascade` was
+    added here so that the lifecycle of `document_sections` is tied
+    to their respective document. We leave the lifecycle of the
+    objects in storage as an exercise to the reader._
```

Comment:
Just a minor reword:

```suggestion
    _Note: Since the video was published, `on delete cascade` was
    added as a new migration so that the lifecycle of `document_sections`
    is tied to their respective document._
```

I removed the `exercise to the reader` part just to prevent confusion. I'm okay with adding an `on delete cascade` on that foreign key too as long as we add a note along with it.

Let's also add the SQL code for the `document_sections` `on delete cascade` migration under this note so that users that are going through this guide step by step can copy and paste that logic right away.